cloud/amazon/ecs_service_facts.py

```diff
@@ -168,19 +170,19 @@ def list_services(self, cluster):
         if cluster and cluster is not None:
             fn_args['cluster'] = cluster
         response = self.ecs.list_services(**fn_args)
-        relevant_response = dict(services = response['serviceArns'])
-        return relevant_response
+        self.module.exit_json(services=response['serviceArns'])
 
     def describe_services(self, cluster, services):
         fn_args = dict()
         if cluster and cluster is not None:
             fn_args['cluster'] = cluster
         fn_args['services']=services.split(",")
         response = self.ecs.describe_services(**fn_args)
-        relevant_response = dict(services = map(self.extract_service_from, response['services']))
+        services = map(self.extract_service_from, response['services'])
+        services_not_running = None
         if 'failures' in response and len(response['failures'])>0:
-            relevant_response['services_not_running'] = response['failures']
-        return relevant_response
+            services_not_running = response['failures']
+        self.module.exit_json(services=services, services_not_running=services_not_running)
```

Comment:
I think that _facts modules should return boto3 return vars as close as possible so i'm not sure about this services= and services_not_running= approach.

Also as per guidelines https://github.com/wimnat/ansible-modules-extras/blob/devel/cloud/amazon/GUIDELINES.md, you should call camel_dict_to_snake_dict to make the results ansible appropriate.